internal/venice-client-common/src/main/java/com/linkedin/venice/httpclient5/HttpClient5Utils.java

```diff
@@ -85,6 +87,25 @@ public HttpClient5Builder setHttp1MaxConnectionsPerRoute(int http1MaxConnections
       return this;
     }
 
+    private ConnectionConfig getDefaultConnectionConfig() {
+      return ConnectionConfig.custom()
+          .setConnectTimeout(CONNECT_TIMEOUT_IN_MINUTES)
+          .setSocketTimeout(CONNECT_TIMEOUT_IN_MINUTES)
+          // Override default value of 2seconds to CONNECTION_INDEFINITE_TIMEOUT
+          .setValidateAfterInactivity(CONNECTION_INDEFINITE_TIMEOUT)
+          .setTimeToLive(CONNECTION_INDEFINITE_TIMEOUT)
```

Comment:
Will there be any side effects to set TTL of connections to -1? In the javadoc it doesn't say if this means it will not enforce TTL or not, only that the default is null.

Reply:
Checked offline and this is good.